docs/includes/protections.md

```diff
@@ -62,9 +62,9 @@ The below example stops trading for all pairs for 4 candles after the last trade
 
 #### MaxDrawdown
 
-`MaxDrawdown` uses all trades within `lookback_period` (in minutes) to determine the maximum drawdown. If the drawdown is below `max_allowed_drawdown`, trading will stop for `stop_duration` (in minutes) after the last trade - assuming that the bot needs some time to let markets recover.
+`MaxDrawdown` uses all trades within `lookback_period` (in minutes or candles) to determine the maximum drawdown. If the drawdown is below `max_allowed_drawdown`, trading will stop for `stop_duration` (in minutes or candles) after the last trade - assuming that the bot needs some time to let markets recover.
```

Comment:
To be honest, i find this confusing now ...
it suggests `lookback_period` can either be minutes or candles - but how does the bot tetermine this?

maybe something like the following?
```
in minutes or candles via `lookback_period_candles`
```

Reply:
Tried to reword it; is it better now?